core/settings/qPrefCloudStorage.cpp

```diff
@@ -77,3 +77,24 @@ HANDLE_PREFERENCE_INT(CloudStorage, "timeout", cloud_timeout);
 HANDLE_PREFERENCE_INT(CloudStorage, "cloud_verification_status", cloud_verification_status);
 
 HANDLE_PREFERENCE_BOOL(CloudStorage, "save_password_local", save_password_local);
+
+QString qPrefCloudStorage::divelogsde_userid()
+{
+	return qPrefPrivate::propValue(keyFromGroupAndName(group, "divelogsde_userid"), "").toString();
```

Comment:
this confuses me - we already have `divelogde_user` and `divelogde_pass` - why create new keys for this?

Reply:
The desktop version uses QSettings directly, so it has a different behavior than the other settings. But of course the names should be identical, corrected.